hpx/runtime/components/server/migration_support.hpp

```diff
@@ -73,42 +141,47 @@ namespace hpx { namespace components
         static threads::thread_function_type
         decorate_action(naming::address::address_type lva, F && f)
         {
-            using util::placeholders::_1;
+            // Make sure we pin the thread at construction of the bound object
```

Comment:
I think this is a type, we don't pin the thread, but the component (object) the task will work on.